.github/workflows/check-sdist.skip.yml

```diff
@@ -0,0 +1,48 @@
+name: Check sdist Skip
+
+# This Workflow is special and contains a workaround for a known limitation of GitHub CI.
+#
+# The problem: We don't want to run the "check sdist" jobs on PRs which contain only changes
+# to the docs, since these jobs take a long time to complete without providing any benefit.
+# We therefore use path-filtering in the workflow triggers for the check sdist jobs, namely
+# "paths-ignore: doc/**". But the "Check sdist post job" is a required job, therefore a PR cannot
+# be merged unless the "Bootstrap post job" completes succesfully, which it doesn't do if we
+# filter it out.
+#
+# The solution: We use a second job with the same name which always returns the exit code 0.
+# The logic implemented for "required" workflows accepts if 1) at least one job with that name
+# runs through, AND 2) If multiple jobs of that name exist, then all jobs of that name have to
+# finish successfully.
```

Comment:
Is this a copy-paste from validate-skip once again? I'd prefer this hack to be documented in one place and referenced at every use site, if possible.

Reply:
That will be addressed in #10359.